src/infinity/libraries/Bridges/LibCMARKCarbon.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import "../LibRetire.sol";
+import "../Token/LibTransfer.sol";
+import "../../interfaces/ICMARKCredit.sol";
+import "../../C.sol";
+
+import "lib/forge-std/src/console.sol";
+
+/**
+ * @author MarcusAurelius
+ * @title LibCMARKCarbon
+ */
+
+library LibCMARKCarbon {
+    event CarbonRetired(

```

Comment:
the CarbonRetired event signature, at least in the subgraph, still requires a pool param, so can be address(0) here. tokenId can be omitted though

```yaml
      eventHandlers:
        - event: CarbonRetired(uint8,indexed address,string,indexed address,string,string,indexed address,address,uint256)
          handler: handleCarbonRetired
        - event: CarbonRetired(uint8,indexed address,string,indexed address,string,string,indexed address,address,uint256,uint256)
          handler: handleCarbonRetiredWithTokenId
          ```

Reply:
yep good catch, added it back into the event